src/Pipeline.h

```diff
@@ -158,6 +158,10 @@ class Pipeline {
      * doing. */
     EXPORT void print_loop_nest();
 
+    /** Write out the loop nests specified by the schedule for this
+     * Function for the specified target. */
+    EXPORT void print_loop_nest(Target &target);
```

Comment:
I don't understand why this exists - for a given schedule, the loop nest structure can't depend on the target.